apps/data-types.md

```diff
@@ -560,6 +560,13 @@ This object is used for the attributes of basePrice, depositPrice, ecoParticipat
 | name | string | The name of the tax class. |
 | links | array of [link](page:apps-data-types#link) | A list of links to the products. |
```

Comment:
add attributes "taxes" (array of taxInfo) to paymentData and shippingData
where taxInfo is an object with these attributes:
 - taxClass: object of taxClassInfo
 - price: object of price

...so you can rename the object "taxes" that you already created to "taxInfo" and update payment and shipping data with the new attribute "taxes"